tracker/cli/db.py

```diff
@@ -0,0 +1,111 @@
+from click import echo
+from click import option
+from click import pass_context
+from flask.cli import with_appcontext
+from flask_migrate.cli import db as db_cli
+
+from tracker import db
+
+
+def abort_if_false(ctx, param, value):
+    if not value:
+        ctx.abort()
+
+
+@db_cli.command()
+@with_appcontext
+def vacuum():
+    """Perform vacuum on the database."""
+
+    echo('Performing database vacuum...', nl=False)
+    db.session.execute('VACUUM')
+    echo('done')
+
+
+@db_cli.command()
+@option('--yes', is_flag=True, callback=abort_if_false,
+        expose_value=False, prompt='Are you sure you want to drop the database?')
+@with_appcontext
+@pass_context
+def drop(ctx):
+    """Drop the database."""
+
+    echo('Dropping database...', nl=False)
+    db.drop_all()
+    echo('done')
+    ctx.invoke(vacuum)
+
+
+@db_cli.command()
+@option('--purge', is_flag=True, help='Purge all data and tables.')
+@with_appcontext
+@pass_context
+def initdb(ctx, purge):
+    """Initialize the database and all tables."""
+
+    if purge:
+        ctx.invoke(drop)
+
+    echo('Initializing database...', nl=False)
+    db.create_all()
+    echo('done')
+
+
+@db_cli.command()
+@option('--integrity/--no-integrity', default=True, help='Check database integrity.')
+@option('--foreign-key/--no-foreign-key', default=True, help='Check foreign keys.')
+@with_appcontext
+def check(integrity, foreign_key):
```

Comment:
Should we add a check to mention that `--no-integrity` + `--no-foreign-key` turns this into a no-op?